packages/google_maps_flutter/google_maps_flutter/ios/Classes/GoogleMapMarkerController.h

```diff
@@ -43,15 +42,19 @@ NS_ASSUME_NONNULL_BEGIN
            registrar:(NSObject<FlutterPluginRegistrar> *)registrar;
 - (void)addMarkers:(NSArray *)markersToAdd;
 - (void)changeMarkers:(NSArray *)markersToChange;
-- (void)removeMarkerIds:(NSArray *)markerIdsToRemove;
-- (BOOL)onMarkerTap:(NSString *)markerId;
-- (void)onMarkerDragStart:(NSString *)markerId coordinate:(CLLocationCoordinate2D)coordinate;
-- (void)onMarkerDragEnd:(NSString *)markerId coordinate:(CLLocationCoordinate2D)coordinate;
-- (void)onMarkerDrag:(NSString *)markerId coordinate:(CLLocationCoordinate2D)coordinate;
-- (void)onInfoWindowTap:(NSString *)markerId;
-- (void)showMarkerInfoWindow:(NSString *)markerId result:(FlutterResult)result;
-- (void)hideMarkerInfoWindow:(NSString *)markerId result:(FlutterResult)result;
-- (void)isMarkerInfoWindowShown:(NSString *)markerId result:(FlutterResult)result;
+- (void)removeMarkersWithIdentifiers:(NSArray *)identifiers;
+- (BOOL)didTapMarkerWithIdentifier:(NSString *)identifier;
+- (void)didStartDraggingMarkerWithIdentifier:(NSString *)identifier
+                                  coordinate:(CLLocationCoordinate2D)coordinate;
```

Comment:
Optional: maybe rename these `coordinate` arguments to `location` or `point`? "Coordinate" feels more like a data type than a description of what it represents. But maybe I don't have enough API context.